### PR TITLE
Added LessonDAO and updated BasicLesson.java and Lesson.java files

### PR DESCRIPTION
**Pull request by Susmita-Rani-Ghosh**

### CODE REVIEW

File: Tecducate/src/main/java/BasicLesson.java
Comment (GeoT82) [inline Tecducate/src/main/java/BasicLesson.java:1]: This is no longer needed

**Review by GeoT82 — COMMENTED** [review]
Need to delete lessonDAO

File: Tecducate/src/main/java/BasicLesson.java
Comment (Susmita-Rani-Ghosh) [reply Tecducate/src/main/java/BasicLesson.java:1]: I deleted the lesson DAO and i need the BasicLesson.java because it's my test file

**Review by GeoT82 — APPROVED** [review]
looks good

**Pull request merged**